hostpath-provisioner.go

```diff
@@ -152,20 +151,13 @@ func main() {
 		klog.Fatalf("Failed to create client: %v", err)
 	}
 
-	// The controller needs to know what the server version is because out-of-tree
-	// provisioners aren't officially supported until 1.5
-	serverVersion, err := clientset.Discovery().ServerVersion()
-	if err != nil {
-		klog.Fatalf("Error getting server version: %v", err)
-	}
-
 	// Create the provisioner: it implements the Provisioner interface expected by
 	// the controller
 	hostPathProvisioner := NewHostPathProvisioner()
 
 	// Start the provision controller which will dynamically provision hostPath
 	// PVs
-	pc := controller.NewProvisionController(clientset, provisionerName, hostPathProvisioner, serverVersion.GitVersion,
+	pc := controller.NewProvisionController(clientset, provisionerName, hostPathProvisioner,
```

Comment:
This is a change with `sig-storage-lib-external-provisioner/v7/controller`, the Kubernetes server version is no longer passed as argument